src/test/java/seedu/address/testutil/PatientBuilder.java

```diff
@@ -217,7 +220,7 @@ public PatientBuilder withNokPhone(String nokPhone) {
      * Builds a patient based on the fields that were set.
      */
     public Patient build() {
-        return new Patient(name, nric, birthdate, sex, phone, email, address, allergy, bloodType,
+        return new Patient(name, nric, birthdate, sex, phone, email, address, allergies, bloodType,
```

Comment:
Thank you for the conscientious effort to ensure that all uses of 'allergy' are changed to 'allergies'. :)